arch/power/crc32_constants.h

```diff
@@ -0,0 +1,1204 @@
+/* Constants table used by crc32_power8.c
+ * Copyright (C) 2021 IBM Corporation
+ *
+ * This file was automatically generated, DO NOT EDIT IT MANUALLY.
+ *
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#define CRC 0x4c11db7
+#define CRC_XOR
+#define REFLECT
+#define MAX_SIZE    32768
+
+#ifndef __ASSEMBLER__
+#ifdef CRC_TABLE
+static const unsigned int crc_table[] = {
```

Comment:
This table appears to be the same as in /crc32_tbl.h.

Reply:
You're right! It's the same as `crc_table[0]`. I'll change the code to use that.

Reply:
The table has been removed in favor of reusing `crc_table[0]` from `crc32_tbl.h`.